frontend/src/components/app/App.js

```diff
@@ -57,6 +59,7 @@ import { useAuthInit } from "./useAuthInit";
 import { useRefreshHook } from "./useRefreshHook";
 import InstrumentsRoute from "../instruments/InstrumentsRoute";
 import { Reports } from "../reports/Reports";
+import { LabworkSamplesPage } from "../management/LabworkSamplesPage";
```

Comment:
labworksamplespage is not a good name for this page... WorkflowAssigmentPage would be more explanatory or WorkflowQueueManagerPage... ?